tree.go

```diff
@@ -683,62 +685,86 @@ func (n *InternalNode) Commit() *Point {
 	internalNodeLevels := make([][]*InternalNode, StemSize)
 	n.fillLevels(internalNodeLevels)
```

Comment:
This diff looks more complicated than it should, I'd suggest looking in split mode.